C-Plus-Plus/cp/ARRAYSUB.cpp

```diff
@@ -0,0 +1,52 @@
+#include<iostream>
```

Comment:
@JayantGoel001 Add a little bit description of the problem and link of the problem.